bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/services/ErrorHandlerUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.jvm.services;
+
+import java.io.PrintStream;
+
+/**
+ * Class contains utility methods for ballerina server error handling.
+ *
+ * @since 0.995.0
+ */
+public class ErrorHandlerUtils {
```

Comment:
Shall we merge this with the BLangVMErrors class it self? Looks like both having some util methods for error handling in JVM?

Reply:
In BVM impl also we had two classes BLangVMErrors and ErrorHandlerUtils. ErrorHandlerUtils class was used in other stdlib modules like http, grpc, websub etc. +1 to move. But I think better to keep  same where migration will be easy. We can move once migration is done. WDYT?

Reply:
ack